lib/shrine/plugins/validate_virus_free.rb

```diff
@@ -9,7 +9,8 @@ module AttacherMethods
         def validate_virus_free(message: nil)
           cached_path = get.download.path
           result = Common::VirusScan.scan(cached_path)
-          # TODO: Log a the full result to sentry
+          # TODO: Log a the full result to ~sentry~ Loki
+          # Logger.info('Common::VirusScan result', result)
```

Comment:
I'd love to go ahead and fix this TODO if we can. What's it take to log to Loki?

Reply:
Set the docker logging driver to loki and point it at the loki server. an example: https://github.com/department-of-veterans-affairs/devops/blob/a04b11a7208379360672ae4dbb24049affd1df1b/ansible/deployment/config/vets-api-server-vagov-dev.yml#L78

Reply:
i think since Loki is live now, as long as I use `Rails.logger`, this should go to Loki. i hope? 🤞🏻